drivers/sensor/adxl372/adxl372_trigger.c

```diff
@@ -51,14 +51,14 @@ static void adxl372_thread_cb(void *arg)
 				     GPIO_INT_EDGE_TO_ACTIVE);
 }
 
-static void adxl372_gpio_callback(struct device *dev,
+static void adxl372_gpio_callback(struct device *dev_port,
```

Comment:
This change is not necessary, it just makes hard to read your patch. Please remove this change everywhere.